main.go

```diff
@@ -26,19 +26,25 @@ func main() {
 		upstream string
 		branch   string
 		username string
-		password string
+		token    string
 	)
 
 	flag.StringVar(&upstream, "upstream", "https://github.com/okkur/gitor.git", "specifies upstream")
 	flag.StringVar(&branch, "branch", "master", "specifies branch")
 	flag.StringVar(&username, "username", username, "specifies username")
-	flag.StringVar(&password, "password", password, "specifies password")
+	flag.StringVar(&token, "token", token, "specifies token/password")
 	flag.Usage = usage
 
 	flag.Parse()
-	command := flag.Args()[0]
-	if command == "update" {
-		update(upstream, branch, username, password)
+
+	switch {
+	case len(os.Args) > 2 && os.Args[2] == "update":
+		command := flag.Args()[0]
+		if command == "update" {
```

Comment:
You are already checking this in the select statement. Try to always ask, why is this specific code necessary.